Graphs/MultiColumnStatusTextGraph/index.js

```diff
@@ -29,7 +29,9 @@ class MultiColumnStatusTextGraph extends TextGraph {
     renderRow = ({data, column, text, color, fontColor, inline, key}) => (
         <div key={key} style={inline ? {} : {display: "block", marginTop: '2%'}}>
             <div style={{width: "25%", display: "inline-block"}}>
-                <div style={{
+                <div 
+                id={`color-${key}`}
```

Comment:
Are you adding id to this element only to use it in test cases? 
Is there any alternate method without modifying the source code?

Reply:
Changes done.